src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,348 @@
+import Image from '../Image/Image'
+import ListItem from '../List/ListItem'
+import _ from 'lodash'
+
+export interface DropdownProps {
+  className?: string
+  fluid?: boolean
+  items?: DropdownListItem[]
+  multiple?: boolean
+  onChange?: (active: DropdownListItem[]) => any
+  placeholder?: string
+  search?: boolean
+  styles?: ComponentSlotStyle<DropdownProps, DropdownState>
+}
+
+export interface DropdownState {
+  active: DropdownListItem[]
+  backspaceDelete: boolean
+  focused: boolean
+  inputValue?: string
+  message?: string
+}
+
+export interface DropdownListItem {
+  key: string
+  header: string
+  content?: string
+  image?: string
+}
+
+/**
+ */
+export default class Dropdown extends UIComponent<Extendable<DropdownProps>, DropdownState> {
+  private inputRef: HTMLElement
+
+  static displayName = 'Dropdown'
+
+  static className = 'ui-dropdown'
+
+  static propTypes = {
+    /** Additional CSS class name(s) to apply.  */
+    className: PropTypes.string,
+
+    /** An input can take the width of its container. */
+    fluid: PropTypes.bool,
+
+    /** Shorthand array of props for ListItem. */
+    items: customPropTypes.collectionShorthand,
+
+    /** A dropdown can have a multiple selection. */
+    multiple: PropTypes.bool,
+
+    /**
+     * Event for request to change 'open' value.
+     * @param {SyntheticEvent} event - React's original SyntheticEvent.
+     * @param {object} data - All props and proposed value.
+     */
+    onChange: PropTypes.func,
+
+    /** A message to serve as placeholder. */
+    placeholder: PropTypes.string,
+
+    /** A dropdown can have a search field. */
+    search: PropTypes.bool,
+
+    /** Additional CSS styles to apply to the component instance.  */
+    styles: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
+  }
+
+  state: DropdownState = {
+    active: this.props.multiple ? [] : null,
+    focused: false,
+    inputValue: '',
+    backspaceDelete: true,
+  }
+
+  public renderComponent({ ElementType, styles, variables }): React.ReactNode {
+    const { search, multiple } = this.props
+    const { inputValue } = this.state
+    const selectedPropIfMultiple = { ...(multiple && { selectedItem: null }) }
+
+    return (
+      <ElementType>
+        <Downshift
+          onChange={this.onDropdownChange}
+          inputValue={inputValue}
+          {...selectedPropIfMultiple}
+          stateReducer={this.stateReducer}
+          itemToString={(item: DropdownListItem) => (item ? item.header : '')}
+        >
+          {({
+            getInputProps,
+            getItemProps,
+            getMenuProps,
+            isOpen,
+            highlightedIndex,
+            selectItemAtIndex,
+            getRootProps,
+          }) => {
+            return (
+              <React.Fragment>
+                <div style={styles.containerDiv} onClick={this.onContainerClick.bind(this, isOpen)}>
+                  <span aria-live="assertive" style={styles.ariaLiveSpan}>
```

Comment:
```aria-*``` attributes as part of accessibility behaviors?

Reply:
created tracking issue for this and will address it after this PR gets in https://github.com/stardust-ui/react/issues/468